### PR TITLE
DCOS-38126 - list of jobs resolver

### PR DESCRIPTION
**Pull request by Fabs**
This PR is about the jobs resolver who allows us to fetch a list of jobs. For this PR to go, we still need some more work to implement `jobListToGraphql` from the work on #3010. 

In order to make it compliant to the Job schema, we needed to do an N + 1 query. We have to fix that soon since it is far from prod ready. https://jira.mesosphere.com/browse/DCOS-38201

Testing this is tricky since it is not connected to any view yet. But you can trust my world on it 😄 or play with the tests to convince yourself.

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?

### CODE REVIEW

File: plugins/jobs/src/js/data/JobModel.ts [hunk 1]
```diff
@@ -289,5 +382,9 @@ export const resolvers = (): IResolvers => ({
 
 export default makeExecutableSchema({
   typeDefs,
-  resolvers: resolvers()
+  resolvers: resolvers({
+    fetchJobs: MetronomeClient.fetchJobs,
+    fetchJobDetail: MetronomeClient.fetchJobDetail,
+    pollingInterval: 2000
```
Comment (Fabs) [inline plugins/jobs/src/js/data/JobModel.ts]: What is the constant that goes here again?

File: plugins/jobs/src/js/data/JobModel.ts [hunk 2]
```diff
@@ -264,22 +267,112 @@ type Query {
...
+  filter: string | null
+): MetronomeClient.JobResponse[] {
+  if (filter === null) {
+    return jobs;
+  }
+
+  return jobs.filter(({ id }) => id.indexOf(filter) !== -1);
+}
+
+function jobListToGraphql(response: MetronomeClient.JobDetailResponse): Job {
```
Comment (Fabs) [inline plugins/jobs/src/js/data/JobModel.ts]: replace that with jobListToGraphql

File: plugins/jobs/src/js/data/JobModel.ts (hunk 1 above)
Comment (DanielMSchmidt) [reply plugins/jobs/src/js/data/JobModel.ts]: `Config.getRefreshRate()`

Comment (Fabs) [reply plugins/jobs/src/js/data/JobModel.ts]: Thanks.

**Pull request closed without merging**